### PR TITLE
EPMCUXDA-6773: Fix entities in order to them match with EURING standart

### PR DESCRIPTION
**Pull request by Shutya**
### What does this PR do?

Fixing entities with EURING codes

### How should I test this?

Run fixtures, start app, see responses, see swagger

### Jira ticket related to this PR

- [EPMCUXDA-EPMCUXDA-6773](https://jira.epam.com/jira/browse/EPMCUXDA-6773)

#### Type of change

- [ ] Adds new features.
- [x] Changes existing functionality.
- [ ] Deprecates features.
- [ ] Removes features.
- [ ] Fixes any bug.
- [ ] Other: security, build process, infrastructure, tests, docs, etc.

### CODE REVIEW

File: src/fixtures/Ring.yml [hunk 1]
```diff
@@ -3,33 +3,33 @@ processor: ./RingProcessor
...
+    sexConcluded: '@sex*'
+    ageMentioned: '@age*'
+    ageConcluded: '@age*'
     status: '@status*'
-    broodsize: '@broodsize*'
+    broodSize: '@broodSize*'
     pullusAge: '@pullusAge*'
     accuracyOfPullusAge: '@accuracyOfPullusAge*'
-    coordinates: '{{address.latitude}} {{address.longitude}}'
+    geographiclCoordinates: '{{address.latitude}} {{address.longitude}}'
```
Comment (mishelen) [inline src/fixtures/Ring.yml]: please, correct typo

File: src/entities/euring-codes/age-entity.ts [hunk 2]
```diff
@@ -18,11 +19,17 @@ export class Age implements Dictionary {
   @Column('varchar', { nullable: true, default: null })
   public desc_byn: string | null;
 
-  @OneToMany(() => Ring, m => m.ageScheme)
-  public ring: Ring[];
+  @OneToMany(() => Ring, m => m.ageMentioned)
+  public ringMentioned: Ring[];
```
Comment (mishelen) [inline src/entities/euring-codes/age-entity.ts]: this naming and the similar below are confusing, would you change it to something like
`mentionedInRing`, `concludedInRing` and so on?

File: src/fixtures/Ring.yml (hunk 1 above)
Comment (Shutya) [reply src/fixtures/Ring.yml]: Thank you, fixed.

File: src/entities/euring-codes/age-entity.ts (hunk 2 above)
Comment (Shutya) [reply src/entities/euring-codes/age-entity.ts]: Yes, I agree with you, it will be more clear. Done.

**Pull request merged**